File: filedata/entity/c#/Basedepartment.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 部门表
    /// </summary>
    [Table("Base_Department")]
    public class Basedepartment
    {
        
        /// <summary>
        /// 主键
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreateTime")]
        public DateTime CreateTime { get; set; }
    
    
        /// <summary>
        /// 创建人Id
        /// </summary>
        [Column("CreatorId")]
        public string CreatorId { get; set; }
    
    
        /// <summary>
        /// 否已删除
        /// </summary>
        [Column("Deleted")]
        public int Deleted { get; set; }
    
    
        /// <summary>
        /// 部门名
        /// </summary>
        [Column("Name")]
        public string Name { get; set; }
    
    
        /// <summary>
        /// 上级部门Id
        /// </summary>
        [Column("ParentId")]
        public string ParentId { get; set; }
    
    
    }
}

File: filedata/entity/java/BaseDepartment.java

package entity;
// 部门表
public class Basedepartment  extends BaseDomain{
    public Basedepartment() {

    }
    
    //部门表
    private String Id;
        
    //部门表
    private Date CreateTime;
        
    //部门表
    private String CreatorId;
        
    //部门表
    private tinyint(4) Deleted;
        
    //部门表
    private String Name;
        
    //部门表
    private String ParentId;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public Date getCreatetime() {
        return Createtime;
    }
    public void setCreatetime(Date Createtime) {
        this.Createtime = Createtime;
    }
            
    public String getCreatorid() {
        return Creatorid;
    }
    public void setCreatorid(String Creatorid) {
        this.Creatorid = Creatorid;
    }
            
    public tinyint(4) getDeleted() {
        return Deleted;
    }
    public void setDeleted(tinyint(4) Deleted) {
        this.Deleted = Deleted;
    }
            
    public String getName() {
        return Name;
    }
    public void setName(String Name) {
        this.Name = Name;
    }
            
    public String getParentid() {
        return Parentid;
    }
    public void setParentid(String Parentid) {
        this.Parentid = Parentid;
    }
             
    @Override
    public String toString() {
        return "Base_Department[Id=" + Id + ",CreateTime=" + CreateTime + ",CreatorId=" + CreatorId + ",Deleted=" + Deleted + ",Name=" + Name + ",ParentId=" + ParentId + "]";
    }
        
    
}
        